### ISSUE TITLE
3.0.0-alpha4 "linkname" does not work for tsget

### ISSUE THREAD

**Issue by rainerjung** [openssl/openssl#12341]
Version: 3.0.0-alpha4

OpenSSL 3 introduced a new build.info feature named "linkname". It is used by the tsget installation procedure to add a symlink from tsget to the installed script tsget.pl. This feature does not work, the symlink does not get installed.

From apps/build.info:

```
  # linkname tells build files that a symbolic link or copy of this script
  # without extension must be installed as well.  Unix or Unix lookalike only.
  SCRIPTS{misc,linkname=tsget}=tsget.pl
```

From configdata.pm:

```
        "scripts" => {
            "apps/CA.pl" => {
                "misc" => "1"
            },
            "apps/tsget.pl" => {
                "linkname=tsget" => "1",
                "misc" => "1"
            },
            "util/shlib_wrap.sh" => {
                "noinst" => "1"
            }
        }
```

From Configurations/unix-Makefile.tmpl:

```
MISC_SCRIPTS={-
        join(" \\\n" . ' ' x 13,
             fill_lines(" ", $COLUMNS - 13,
                        map { my $x = $unified_info{attributes}->{scripts}->{$_}->{linkname};
                              $x ? "$_:$x" : $_ }
                        grep { !$unified_info{attributes}->{scripts}->{$_}->{noinst}
                               && $unified_info{attributes}->{scripts}->{$_}->{misc} }
                        @{$unified_info{scripts}}))
-}
```

From Makefile:

```
MISC_SCRIPTS=apps/CA.pl apps/tsget.pl
```

The correct line should have been:

```
MISC_SCRIPTS=apps/CA.pl apps/tsget.pl:tsget
```

(Added ":tsget").

It seems the additional token separator "=" in build.info "linkname=tsget" isn't corectly handled by the tokenizer?

Thanks and regards,

Rainer

**Comment by levitte** [comment]
Fixed!  See #12344

**Comment by rainerjung** [comment]
Confirmed, works for me.
Thanks!!
